.github/workflows/release.yml

```diff
@@ -97,6 +112,14 @@ jobs:
         run: task build
         if: matrix.operating-system == 'macos-10.15'
 
+      - name: Create autoupdate files
+        run: ${GOPATH}/bin/go-selfupdate arduino-create-agent${{ matrix.ext }} ${TAG_VERSION} # this will create `public/` dir with compressed full bin (<version>/<os>-<arch>.gz) and a json file
+        if: steps.prerelease.outputs.IS_PRE != 'true'
```

Comment:
![image](https://user-images.githubusercontent.com/34278123/103420234-8d232380-4b96-11eb-8a80-5c6e2adcbc9e.png)
The json file contains:
```
{
    "Version": "1.2.0",
    "Sha256": "rm8UXVGaGsh8KCVsG0JYBK9VoULaBu2zA/aVPuRf1Qo="
}
```
and `public/1.2.0/linux-amd64.gz` is the archive containing the **full** binary used for the updating